Makefile

```diff
@@ -1,9 +1,17 @@
+export PATH := $(CURDIR)/.venv/bin:$(PATH)
+
 build:
 	@yarn turbo run build
 build-sequential:
 	@yarn turbo run --concurrency 1 build
-python-deps:
-	@python3 -m pip install -r requirements.txt --root-user-action=ignore
+
+venv-setup:
```

Comment:
Do we have issues to link to explaining why we have to use the system Python?

Reply:
Added https://github.com/PrairieLearn/PrairieLearn/pull/12253/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R8